src/components/stegvisning/AFP/AFPFoedtEtter1963/AFP.tsx

```diff
@@ -24,10 +28,19 @@ interface Props {
 export function AFP({ previousAfp, onCancel, onPrevious, onNext }: Props) {
   const intl = useIntl()
 
+  const foedselsdato = useAppSelector(selectFoedselsdato)
+  const foedtEtter1963 = isFoedtEtter1963(foedselsdato)
+  const hasErApotekerError = useAppSelector(selectHasErApotekerError)
+
   const [validationError, setValidationError] = React.useState<string>('')
   const [showVetIkkeAlert, setShowVetIkkeAlert] = React.useState<boolean>(
     previousAfp === 'vet_ikke'
   )
+  const [showApotekerAlert, setShowApotekerAlert] = React.useState<boolean>(
+    Boolean(
```

Comment:
Redundant `Boolean` bruk her siden Logical `AND` og `OR` altid evaluerer til `boolean`
```suggestion
    previousAfp === 'ja_offentlig' && foedtEtter1963 && hasErApotekerError
```